src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/domain/entities/AsylumCaseDefinition.java

```diff
@@ -35,6 +35,8 @@
         "homeOfficeReferenceNumber", new TypeReference<String>(){}),
     PAYMENT_REFERENCE(
         "paymentReference", new TypeReference<String>(){}),
+    PAYMENT_DATE(
+        "paymentDate", new TypeReference<String>(){}),
     ERROR_CODE(
```

Comment:
Should be "PAYMENT_ERROR_CODE" and "PAYMENT_ERROR_MESSAGE" - not sure if needs rebasing?

Reply:
Changed